timescaledb.rb

```diff
@@ -39,6 +40,12 @@ def install
     (share/"timescaledb").install Dir["stage/**/share/postgresql*/extension/*"]
   end
 
+  def check_postgresql_version
+    if postgresql.version >= Version.new('17.0') && postgresql.revision < 3
+      odie "PostgreSQL 17.03 or higher is required, but you have #{postgresql.version}.#{postgresql.revision}"
```

Comment:
Actually those are the bad versions: https://x.com/michaelfreedman/status/1857148280167174455

Reply:
Let's just double check my understanding:

* 17.0 is still working
* 17.01 does not work
* 17.02 or greater it's back being compatible - Checking release notes of [17.2] I see the  line:

> Repair ABI break for extensions that work with struct ResultRelInfo (Tom Lane) [§](https://postgr.es/c/6bfacd368)

https://git.postgresql.org/gitweb/?p=postgresql.git;a=commitdiff;h=6bfacd368